### ISSUE TITLE
Unexpected token o in JSON at position 1

### ISSUE THREAD

**Issue by Trung0246** [kickscondor/fraidycat#215]
**Before you start.**

* [x] I have checked the [Common Problems](https://github.com/kickscondor/fraidycat/wiki/Common-Problems) page.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Vivaldi
 - Version: 1.1.7
 - URL you are attempting to follow [if applicable]

**Describe the bug**
Apparently the extension attempts to parse a raw object and throw a bunch of parse error while calling JSON.parse

![image](https://user-images.githubusercontent.com/11626920/113391490-7385a980-9348-11eb-8c49-b5a6cda0e03d.png)
![image](https://user-images.githubusercontent.com/11626920/113391614-a760cf00-9348-11eb-8050-24b54ecaff70.png)

Any idea what's going on?

**Comment by kickscondor** [comment]
Yes - so this was a bug in the feed-sensing code. I submitted an update last night to fix this - is out in Firefox, should be out in Chrome soon.
